### PR TITLE
[Authentication][Signoutflowadded][1570772]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Added section for sign out flow with code for issue 424. Please review.

### CODE REVIEW

File: msteams-platform/tabs/how-to/authentication/auth-silent-aad.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ if (loginHint) {
 
 ### Authenticate
 
-If ADAL has an unexpired token cached for the user, use the token. Alternately, attempt to get a token silently by calling `acquireToken(resource, callback)`. ADAL.js will call your callback function with the requested token, or give an error if authentication fails.
+If ADAL has an unexpired token cached for the user, use the token. Alternately, attempt to get a token silently by calling `acquireToken(resource, callback)`. ADAL.js calls your callback function with the requested token, or gives an error if authentication fails.
```
Comment (KirtiPereira) [inline msteams-platform/tabs/how-to/authentication/auth-silent-aad.md:64]:
```suggestion
If ADAL has a token cached for the user that has not expired, use that token. Alternately, get a token silently by calling `acquireToken(resource, callback)`. ADAL.js calls the callback function with the requested token, or gives an error if authentication fails.
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@v-rodriguezd - please validate and incorporate the suggestions.

File: msteams-platform/tabs/how-to/authentication/auth-silent-aad.md [hunk 2]
```diff
@@ -112,3 +112,14 @@ if (authContext.isCallback(window.location.hash)) {
     }
 }
 ```
+
+### Handle sign out flow
+
+Use the following code to handle sign out flow in AAD Auth:
```
Comment (Trinetra-MSFT) [inline msteams-platform/tabs/how-to/authentication/auth-silent-aad.md:118]: Please give some insight on sign out flow, "While we do logout for teams tab or Bot, we clear the current session"

File: msteams-platform/tabs/how-to/authentication/auth-silent-aad.md [hunk 3]
```diff
@@ -112,3 +112,14 @@ if (authContext.isCallback(window.location.hash)) {
...
+
+```javascript
+function logoutV1() {
```
Comment (Trinetra-MSFT) [inline msteams-platform/tabs/how-to/authentication/auth-silent-aad.md:121]: Please rename this Method name to "logout" remove V1

File: msteams-platform/tabs/how-to/authentication/auth-silent-aad.md [hunk 4]
```diff
@@ -112,3 +112,14 @@ if (authContext.isCallback(window.location.hash)) {
...
+localStorage.clear();
+window.location.href = "@Url.Action("ChooseAuth", "Home")";
```
Comment (Trinetra-MSFT) [inline msteams-platform/tabs/how-to/authentication/auth-silent-aad.md:123]: Please replace chooseAuth with `<<Action Name>>` and Home to `<<Controller Name>>`

**Review by Trinetra-MSFT — CHANGES_REQUESTED** [review]

**Review by Trinetra-MSFT — APPROVED** [review]

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR

**Pull request merged**